src/components/dialog/dialog.tsx

```diff
@@ -202,6 +195,15 @@ export class Dialog {
         }
     }
 
+    @Watch('closingActions')
+    protected closingActionsChanged(newValue: boolean, oldValue: boolean) {
+        if (isEqual(newValue, oldValue)) {
```

Comment:
I don't think we need to use `lodash` just to compare two `boolean`s :smile: 
```suggestion
        if (newValue === oldValue) {
```

Reply:
I don't think the typings of `newValue` and `oldValue` are correct here:

<img width="545" alt="Screenshot of the diff showing that the property closingActions is an object with two boolean properties, not a boolean itself" src="https://user-images.githubusercontent.com/154725/163330375-1521e5a8-ed52-4e6f-bc30-c477442b9ce0.png">

Reply:
You are right, I will do a fixup! 🙂